src/api/service/Auth.service.ts

```diff
@@ -14,50 +21,47 @@ const prisma = new PrismaClient();
 export class AuthService {
   constructor() {}
   //TO-DO: finish the doctors and admin login in order to correctly take the output of the login method.
```

Comment:
is this To-Do done?

Reply:
no, it has not been done yet. keep being a TO-DO

Reply:
ok, thx 4 clarifying.